### DISCUSSION TITLE
How to use AddOns?

### DISCUSSION THREAD

**Discussion by jfmatth** [kubero-dev/kubero#109] [Q&A]
Looking to add a postgres DB to my python app, but haven't a clue on how to use the addon?

**Comment by mms-gianni** [comment]
Create the addons with the UI. 

Use the created credentials to connect to the PostgreSQL service address. 

You will need to add the env variables manually for now. Working on adding them automatically for every addon. 

<img width="1220" alt="image" src="https://user-images.githubusercontent.com/2052196/222427252-d84424c6-f8fe-4878-94e0-a20b00ce05aa.png">

<img width="941" alt="image" src="https://user-images.githubusercontent.com/2052196/222427675-8d6791a1-fec0-4b11-9b6c-f67c7b7aecdf.png">

<img width="618" alt="image" src="https://user-images.githubusercontent.com/2052196/222427892-074087d9-87da-4aa6-908e-cf86fdf4a3dc.png">

So in my case i will need to add the following ENV vars: 

```
PGHOST=test-postgresql
PGUSER=test
PGDATABASE=postgresql
PGPASSWORD=test
PGPORT=5432
```

**Reply by celelui** [reply]
currently something exist to bind secrets instead of env var ?
i mean is it possible to have vault(or similar) -> secretOperator -> App
